libkbfs/key_manager.go

```diff
@@ -446,24 +446,22 @@ func (km *KeyManagerStandard) Rekey(ctx context.Context, md *RootMetadata, promp
 		if err != nil {
 			return false, nil, err
 		}
-		if latestHandle == nil {
-			return false, nil, NoSuchTlfHandleError{md.ID}
-		}
-		if latestHandle.ConflictInfo != resolvedHandle.ConflictInfo {
+		resolvedInfo := resolvedHandle.ConflictInfo()
+		if latestHandle.ConflictInfo != resolvedInfo {
```

Comment:
This is a pointer comparison. Is this really correct? I would have expected this to use `CodecEqual()`. @aalness